resources/options_lookup.tsv

```diff
@@ -102,14 +102,14 @@ Bedrooms	2	ResStockArguments	geometry_unit_num_bedrooms=2	geometry_unit_num_bath
 Bedrooms	3	ResStockArguments	geometry_unit_num_bedrooms=3	geometry_unit_num_bathrooms=auto
 Bedrooms	4	ResStockArguments	geometry_unit_num_bedrooms=4	geometry_unit_num_bathrooms=auto
 Bedrooms	5	ResStockArguments	geometry_unit_num_bedrooms=5	geometry_unit_num_bathrooms=auto
-Building America Climate Zone	Cold
-Building America Climate Zone	Hot-Dry
-Building America Climate Zone	Hot-Humid
-Building America Climate Zone	Marine
-Building America Climate Zone	Mixed-Dry
-Building America Climate Zone	Mixed-Humid
-Building America Climate Zone	Subarctic
-Building America Climate Zone	Very Cold
+Building America Climate Zone	Cold	ResStockArguments	site_soil_type=unknown	site_moisture_type=mixed
```

Comment:
This is just an example of using some of the new GHP-related arguments. We'd likely be using newly created TSV file(s) for assigning GHP-related arguments.